tools/make_block_list.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
What is the purpose of this file?

Reply:
This is the file I've been using to generate the .block_list file for the scalable output (forgot I had this file in here). Scalable output doesn't currently output the .block_list file, which is what's given to YT to load the dataset. I'll post an issue for this.